Workbooks/Images/Logos/illusive_logo_workbook.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="126px" height="180px" viewBox="0 0 126 180" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Please remove xmlns:xlink="http://www.w3.org/1999/xlink" as it interferes with rendering logo on Azure portal.

Reply:
Fixed